### ISSUE TITLE
Can I access the trajectory object polynomial coefficients?

### ISSUE THREAD

**Issue by guri-dominic** [pantor/ruckig#144]
Your package is great! Easy to install and use.
After solving for the trajectory, is it possible to get the underlying polynomial coefficients? Or the times at which the trajectory switches from one section polynomial to another?

Can you also explain how `return_section=True` is supposed to work? It seems to be returning the acceleration value.

`Trajectory.at_time(self: ruckig.Trajectory, time: float, return_section: bool = False) -> tuple`

**Comment by pantor** [comment]
Yeah, if you're using Python you can compile Ruckig with the `WITH_EXPOSE_INTERNAL=ON` cmake flag. Then, you can access all trajectory coefficients via e.g. `trajectory.profiles[i][dof].j` for the jerk and similarly for `t`, `a`, `v`, `p` for times and accelerations, velocities and positions. The relevant source code is [here](https://github.com/pantor/ruckig/blob/612eb3cb6bc8699c48cf49170d136dfedb469ab2/src/ruckig/python.cpp#L163).
